frappe/website/doctype/web_form/web_form.py

```diff
@@ -365,6 +365,7 @@ def has_web_form_permission(self, doctype, name, ptype='read'):
 
 
 @frappe.whitelist(allow_guest=True)
+@rate_limit(key='web_form', limit=5, seconds=60, methods=['POST'])
```

Comment:
should we make it configurable?
eg:
![image](https://user-images.githubusercontent.com/28212972/131956773-a7c47e96-8b8d-4dad-b2a6-3bbfac10e5f8.png)

Reply:
Yes we can